plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSinkProvider.java

```diff
@@ -184,10 +188,19 @@ private DeltaLakeCdfPageSink createCdfPageSink(
                         metadata.getPhysicalColumnType(),
                         partitionKeys.contains(metadata.getName()) ? PARTITION_KEY : REGULAR))
                 .collect(toImmutableList());
+        ImmutableList.Builder<DeltaLakeColumnHandle> cdfColumns = ImmutableList.builder();
+        cdfColumns.addAll(tableColumns);
+        cdfColumns.add(new DeltaLakeColumnHandle(
```

Comment:
nit: We don't need to delay building the list now:
```java
        List<DeltaLakeColumnHandle> cdfColumns = ImmutableList.<DeltaLakeColumnHandle>builder()
                .addAll(tableColumns)
                .add(new DeltaLakeColumnHandle(
                        CHANGE_TYPE_COLUMN_NAME,
                        VARCHAR,
                        OptionalInt.empty(),
                        CHANGE_TYPE_COLUMN_NAME,
                        VARCHAR,
                        REGULAR))
                .build();
```

Reply:
Thank you.
I changed `cdfColumns` back to `allColumns` to avoid using abbreviations.